apps/acrs/sections/summary-data-sections.js

```diff
@@ -115,6 +115,37 @@ module.exports = {
       }
     ]
   },
+  'family-in-your-referral': {
+    steps: [
+      {
+        steps: '/parent',
+        field: 'parent',
+        parse: (list, req) => {
+          if (!req.sessionModel.get('steps').includes('/parent')) {
+            return null;
+          }
+          return req.sessionModel.get('parent') === 'yes' ?
+            'Yes' : 'No';
+        }
+      },
+      {
+        step: '/parent-summary',
+        field: 'referred-parents',
+        parse: (list, req) => {
+          if (req.sessionModel.get('referred-parents') === undefined ||
+            req.sessionModel.get('referred-parents').aggregatedValues.length < 1) {
+            return null;
+          }
+          if (list.aggregatedValues.length) {
```

Comment:
Alternatively if `list` isn't the correct aggregation array should a new list var be populated from session here?

Reply:
As above